### DISCUSSION TITLE
ATTENTION

### DISCUSSION THREAD

**Discussion by Voiceofreasonwow** [chromiecraft/chromiecraft#3172] [General]
I HAVE an ISSUE

**Comment by Evelmike** [comment]
> I HAVE an ISSUE

Care to elaborate a bit?

**Comment by Voiceofreasonwow** [comment]
I cannot log on to my character

**Reply by 55Honey** [reply]
@Voiceofreasonwow 
We'd love to help you, but we're going to need details about your character/account/setup to be able to help at all.

If you can, please join our discord and make a #support-ticket, so you don't need to share your personal data here in this public channel.

Otherwise please tell us what *exactly* do you try? And what *exactly* you see when you do it? Thanks.

**Comment by Voiceofreasonwow** [comment]
get bent
